deploy/openj9-chart/README.md

```diff
@@ -0,0 +1,106 @@
+# OpenJ9 Helm Chart
+
+## Introduction
+
+Eclipse OpenJ9 is an independent implementation of a Java Virtual Machine.
+
+The OpenJ9 Chart allows you to deploy and manage java applications running on OpenJ9 into OKD or OpenShift clusters. You can also perform Day-2 operations such as rolling update your application, enabling JITServer technology for continued performance enhancements or gathering JVM dumps using the operator.
+
+## Resources Required
+
+### System
+
+* CPU Requested : 1 CPU
+* Memory Requested : 256 MB
+
+### Storage
+
+* Currently no storage or volume needed.
+
+## Chart Details
+
+* Install one `deployment` running Adopt OpenJ9 image
+* Install one `service` that is JITServer technology enabled by default
+
+## Prerequisites
+
+### Red Hat OpenShift SecurityContextConstraints Requirements
+
+This chart does not require SecurityContextConstraints to be bound to the target namespace prior to installation.
+
+### PodSecurityPolicy Requirements
+
+This chart does not require PodSecurityPolicy to be bound to the target namespace prior to installation.
+
+### PodDisruptionBudgets Requirement
+
+This chart does not require PodDisruptionBudgets to be bound to the target namespace prior to installation.
+
+## Installing the Chart
+
+``` bash
+helm install my-release PATH_TO_OPENJ9_CHART
+```
+
+This chart installs OpenJ9 instance for Java 14 on x86 platform by default. If you would like to use OpenJ9 chart on other platform and different Java version, please override keyword `arch` or `image` with `--set` option.
+
+## Verifying the Chart
+
+``` bash
+helm test my-release
+helm status my-release
+```
+
+## Uninstalling the Chart
+
+``` bash
+helm delete my-release
+```
+
+## Getting started with OpenJ9 JITServer Technology
+
+JITServer Technology can be used for any OpenJ9 OpenJDK based Java applications. Please make sure that JITServer image runs the same build version of JDK as your java application image.
+
+``` bash
+helm install my-release PATH_TO_OPENJ9_CHART
+```
+
+Once JITServer is deployed, you can enable JITServer by adding the following JVM option to your Java application.
+
+You may change `<servicename>` and `<serverport>` by editing `values.yaml`. Look for service name with `kubectl get service`, `<serverport>` is set to `38400` by default.
+
+``` bash
+IBM_JAVA_OPTIONS = -XX:+UseJITServer -XX:JITServerAddress=<servicename> -XX:JITServerPort=<serverport>
```

Comment:
IBM_JAVA_OPTIONS is not an environment variable used by OpenJ9 - OPENJ9_JAVA_OPTIONS is the equivalent.

Reply:
Thanks, this has been fixed in the latest commit.